b-lab-org/ember-cli-impact-core
# [Support] Updating the `question-card` component to show points available and points earned [#144067753]

#### Overview
This is the UI update to support showing points available and points earned.

https://www.pivotaltracker.com/story/show/144067753

#### Notes
- I took the approach here of having the component be "dumb", in that the consuming applications determine when or how to show points.

#### How to test
See impact-manage and impact-app PRs.

Additionally, you can QA the style change in the dummy app on the question-cards route.
